CONTRIBUTING.adoc

```diff
@@ -64,6 +64,9 @@ The commands are:
 start:android   Start the Android emulator
 start:ios       Start the iOS emulator
 
+dev:android     Start Android emulator with TypeScript Complier
```

Comment:
Personally, I'd name these `start:*`, and rename the existing `start:*` to `emulator:*` or `run:*`